content/en/posts/js/introduction-a-reactjs/index.md

```diff
@@ -0,0 +1,176 @@
+---
+date: "2014-06-04"
+route: /posts/js/introduction-a-reactjs/
```

Comment:
I would go for "content/en/posts/js/introduction-TO-reactjs/index.md" (in lowercase)